custom_components/philips_airpurifier_http/const.py

```diff
@@ -0,0 +1,144 @@
+""" philips_airpurifier constants"""
```

Comment:
Please use `git mv` so git recognizes this file has moved instead of adding it as a new one.

Reply:
hope i managed to change this correctly